### PR TITLE
add avalon.js 1.4.4

### PR DESCRIPTION
**Pull request by zhangchunlin**
avalon.js : https://github.com/RubyLouvre/avalon
#4703 abandoned

### CODE REVIEW

File: ajax/libs/avalon.js/package.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    "name": "RubyLouvre(司徒正美) and other contributors",
+    "url": "http://home.cnblogs.com/u/rubylouvre/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RubyLouvre/avalon.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/RubyLouvre/avalon.git",
```
Comment (PeterDaveHello) [inline ajax/libs/avalon.js/package.json]: Should use git protocol here, thanks.

**Comment by zhangchunlin** [comment]
@PeterDaveHello fixed now, pls check again.

**Comment by zhangchunlin** [comment]
@RubyLouvre avalonjs has been submitted to cdnjs.
@PeterDaveHello Will cdnjs do the autoupdate automatically? I am asking if I should do manually when new avalonjs version released?

**Comment by PeterDaveHello** [comment]
Will do the autoupdate automatically.

**Comment by zhangchunlin** [comment]
Thanks!

**Comment by PeterDaveHello** [comment]
np

**Pull request merged**